src/provenance/LocalStorageProvenanceGraphManager.ts

```diff
@@ -22,12 +22,20 @@ export default class LocalStorageProvenanceGraphManager implements IProvenanceGr
     mixin(this.options, options);
   }
 
+  private loadFromLocalStorage(suffix: string, defaultValue: string) {
+    try {
+      return JSON.parse(this.options.storage.getItem(this.options.prefix + suffix) || defaultValue);
```

Comment:
instead of this `||` please use a proper check otherwise maybe valid empty strings are not detected